components/Cards/DetailsCard.js

```diff
@@ -35,19 +47,135 @@ export default function DetailsCard({ id, isFavorite, toggleFavorite, place }) {
 
   const [comments, setComments] = useState([]);
   const [editingCommentId, setEditingCommentId] = useState(null);
+  const [editingPlaceId, setEditingPlaceId] = useState(false);
```

Comment:
the name of the state suggests to me that an ID / string is being stored here and not a boolean value.

Variables / States that store a boolean value are usually prefixed with `is`, `has`, `can`